client/landing/stepper/index.tsx

```diff
@@ -69,11 +68,7 @@ const getSiteIdFromURL = () => {
 	return siteId ? Number( siteId ) : null;
 };
 
-const HOTJAR_ENABLED_FLOWS = [
-	SITE_MIGRATION_FLOW,
-	HOSTED_SITE_MIGRATION_FLOW,
-	MIGRATION_SIGNUP_FLOW,
-];
+const HOTJAR_ENABLED_FLOWS = [ SITE_MIGRATION_FLOW, HOSTED_SITE_MIGRATION_FLOW ];
```

Comment:
We should work out whether we should keep HotJar enabled for the other flows, but that should be separate follow-up work.

Reply:
Created this issue to tackle follow up cleanup: https://github.com/Automattic/wp-calypso/issues/101586